lib/krikri/aggregation_entity_behavior.rb

```diff
@@ -0,0 +1,14 @@
+
+# TODO:  move to a subdirectory?
+
+module Krikri
+  class AggregationEntityBehavior < EntityBehavior
```

Comment:
I was taking a cue from some of the behaviors in the `harvesters` directory, where there aren't namespaces, like `OAISkipDeletedBehavior < BasicSaveBehavior` but I am happy to add the namespace.